main.cpp

```diff
@@ -65,10 +71,37 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int temperature = 20;
+    int phaseShift = -90;
+    int voltage = 15;
+
+    unsigned int age = 5;
+    unsigned int rms = 95;
+    unsigned int halfLife = 30;
+
+    char switch1State = 2;
+    char modeSelector = 9;
+    char midiChannel = 16; 
+    /*
+    tried char note = "a"; but got this error : 
+    
+    main.cpp:84:10:      error: cannot initialize a variable of type 'char' with an lvalue of type
+      'const char [2]'
+    char note = "a";
+    */
+    float pot1 = 0;
+    float filterPosition = 25.25f;
+    float modulationRange = -55.2f;
+
+    bool led1 = false; //led1 output pin at low level
+    bool siren = true;
+    bool sequencerRunning = false;
+
+    double jeffBezosWealth = 160000000000;
+    double worldwideAntPopulation = 0;
+    double countdown = 0; 
```

Comment:
`double`'s end with `.` and a decimal number.  
`double worldWideAntPopulation = 0.0;` 
`double pi = 3.14;`
- [ ] revise.